### PR TITLE
chipsInput add onFocus callback

### PR DESCRIPTION
**Pull request by vidmantas0010**
## Description
*ChipsInput add onFocus callback prop*

## Changelog
*chipsInput*

### CODE REVIEW

File: src/components/chipsInput/index.tsx [hunk 1]
```diff
@@ -298,6 +298,7 @@ class ChipsInput extends Component<OwnProps, State> {
   }
 
   onFocus = () => {
+    _.invoke(this.props, 'onFocus');
```
Comment (Inbal-Tish) [inline src/components/chipsInput/index.tsx:301]: Please use optional chaining and pass the event (i.e. e: NativeSyntheticEvent<TextInputFocusEventData>)

Comment (Inbal-Tish) [reply src/components/chipsInput/index.tsx:301]: If you can do the same for 'onBlur' that will be much appreciated!

**Comment by Inbal-Tish** [comment]
@vidmantas0010 Merge master to your branch (will fix failed build)

File: src/components/chipsInput/index.tsx (hunk 1 above)
Comment (vidmantas0010) [reply src/components/chipsInput/index.tsx]: I can't use optional chaining, because onKeyPress above is using lodash invoke.

Comment (Inbal-Tish) [reply src/components/chipsInput/index.tsx]: The type should be `NativeSyntheticEvent<TextInputFocusEventData>` otherwise it fails on TS error

**Comment by Inbal-Tish** [comment]
@vidmantas0010 Pull master again, we had tests failing that broke the builds

**Comment by ethanshar** [comment]
Closing this. 
We changed our implementation, we're now using `Incubator.ChipsInput` internally. 
@vidmantas0010 This should now work for you

**Pull request closed without merging**